File: bootstrap/roles/testbench/files/code/sample.go
package main

import "fmt"

const Max = 10

type User struct {
	ID   int
	Name string
}

func (u *User) Greet(times int) string {
	// TODO: builtins, operators
	out := ""
	for i := 0; i < times && i < Max; i++ {
		out += fmt.Sprintf("hi %s\n", u.Name)
	}
	return out
}

File: bootstrap/roles/testbench/files/code/sample.rs
//! Module doc comment.
use std::collections::HashMap;

const MAX: u32 = 10;

#[derive(Debug)]
struct User<'a> {
    id: u64,
    name: &'a str,
}

impl<'a> User<'a> {
    fn greet(&self, times: usize) -> String {
        // TODO: builtins, operators
        let mut out = String::new();
        for _ in 0..times.min(MAX as usize) {
            out.push_str(&format!("hi {}\n", self.name));
        }
        out
    }
}
